frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -135,15 +155,27 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
               onExit={() => setFindMoreOpen(false)}
             />
             <div className="vl"></div>
-            <Col className="bar-item">
-              <PropertyFilterOptions disabled={findMoreOpen} />
-            </Col>
+
+            <AgencyCol>
+              {showAllAgencySelect ? (
+                <PropertyFilterAgencyOptions disabled={findMoreOpen} agencies={agencies} />
+              ) : (
+                <ParentSelect
+                  field="agencies"
+                  options={agencies}
+                  filterBy={['code', 'label', 'parent']}
+                  placeholder="Agency"
+                  selectClosest
+                  disabled={findMoreOpen}
+                />
+              )}
+            </AgencyCol>
             <Col className="map-filter-typeahead">
               <AsyncTypeahead
-                disabled={findMoreOpen}
+                disabled={findMoreOpen || values.includeAllProperties === true}
```

Comment:
I don't think this is correct.  Can SRES search for properties within a given agency still?  `admin-properties` should always be able to filter.

Reply:
updating.